test/test.js

```diff
@@ -46,6 +46,44 @@ it('thenifyAll(fs, destination)', function () {
   })
 })
 
+it('thenifyAll(user, ["show"]) should maintain the receiver', function () {
+  var user = Object.create({
```

Comment:
ohhh i see waht you're doing. what you should be doing instead of `var user = Object.create(thenify({}))`.